### PR TITLE
Fix typo in variable DEAULT_FLOW_RESOURCES_FOLDER

### PR DESCRIPTION
**Pull request by Haprog**
Replace DEAULT_FLOW_RESOURCES_FOLDER with DEFAULT_FLOW_RESOURCES_FOLDER, but keep the old variable as deprecated.

### CODE REVIEW

File: flow-maven-plugin/src/main/java/com/vaadin/flow/plugin/maven/FlowModeAbstractMojo.java [hunk 1]
```diff
@@ -25,7 +25,6 @@
 import com.vaadin.flow.server.Constants;
 
 import static com.vaadin.flow.server.Constants.VAADIN_SERVLET_RESOURCES;
-import static com.vaadin.flow.server.frontend.FrontendUtils.DEAULT_FLOW_RESOURCES_FOLDER;
```
Comment (Haprog) [inline flow-maven-plugin/src/main/java/com/vaadin/flow/plugin/maven/FlowModeAbstractMojo.java:28]: This was an unused import.

File: flow-server/src/main/java/com/vaadin/flow/server/frontend/FrontendUtils.java [hunk 2]
```diff
@@ -128,10 +128,21 @@
     /**
      * Default folder for copying front-end resources present in the classpath
      * jars.
+     * @deprecated This is deprecated due to a typo.
+     *             Use DEFAULT_FLOW_RESOURCES_FOLDER instead.
+     * @see #DEFAULT_FLOW_RESOURCES_FOLDER
      */
+    @Deprecated
     public static final String DEAULT_FLOW_RESOURCES_FOLDER = TARGET
             + "flow-frontend";
```
Comment (joheriks) [inline flow-server/src/main/java/com/vaadin/flow/server/frontend/FrontendUtils.java:137]: Any particular reason for duplicating the value of the constant? Would be less redundancy to have the new `DEFAULT_FLOW_RESOURCES_FOLDER` introduced first with value and then the deprecated name backreferencing it:
```java
    @Deprecated
    public static final String DEAULT_FLOW_RESOURCES_FOLDER = DEFAULT_FLOW_RESOURCES_FOLDER;

```

**Review by joheriks — CHANGES_REQUESTED** [review]

File: flow-server/src/main/java/com/vaadin/flow/server/frontend/FrontendUtils.java (hunk 2 above)
Comment (Haprog) [reply flow-server/src/main/java/com/vaadin/flow/server/frontend/FrontendUtils.java:137]: Good point. I can change it (maybe later today when I'm back at my computer, or tomorrow)

Comment (Haprog) [reply flow-server/src/main/java/com/vaadin/flow/server/frontend/FrontendUtils.java:133]: Done.

**Review by joheriks — APPROVED** [review]

**Pull request merged**